src/validation.ts

```diff
@@ -833,8 +833,9 @@ export const appSyncSchema = {
       ],
       errorMessage: 'contains invalid pipeline function definitions',
     },
+    apiId: { $ref: '#/definitions/stringOrIntrinsicFunction' },
   },
-  required: ['name', 'authentication'],
+  required: ['name'],
```

Comment:
as mentioned above, we should have 2 schemas: one for a usual appsync config (main api or single stack) and one for "external API" which only accepts supporters fields.
This way, `authentication` can remain required in single stack scenarios or for main stack and be "forbidden" for sub stacks